gtnh-modpack.json

```diff
@@ -872,7 +872,7 @@
       "filename": "lootgames-1.7.10-2.0.5.jar",
       "download_url": "https://api.github.com/repos/GTNewHorizons/LootGames/releases/assets/56320179",
       "browser_download_url": "https://github.com/GTNewHorizons/LootGames/releases/download/2.0.5/lootgames-1.7.10-2.0.5.jar",
-      "license": "ARR for people outside of the NH organization, might need some clarifications",
+      "license": "All Rights Reserved for people outside of the NH organization, might need some clarifications, pending community outreach.",
```

Comment:
Lootgames was originally developped by Namikon (one of the NH admins) and is now maintained by TimeConqueror. I don't see why we need pending community outreach for this one?

Reply:
Was listed only because the "might need clarifications" text listed previously, which would tend to indicate some ambiguity requiring resolution.

Reply:
maybe remove it, as it's pretty clear in the license file of the repo